CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+# 1.0.10-rc1 (2020-06-26)
+## Edge Agent
+### Features
+* Update codebase to dotnet 3.1. [f87a18a](https://github.com/Azure/iotedge/commit/f87a18a487ea0c05752254aaba04a4f89028120a)
+* Install Trust Bundle. [4f85dcc](https://github.com/Azure/iotedge/commit/4f85dcc7d3fd6d4772d4a9b86ec3ecad651938fd)
+* Add metrics upload to IoTHub feature [eff5c85](https://github.com/Azure/iotedge/commit/eff5c859e101d14eda85b80513136a2f3a473892)
+* Add "Cmd", "Entrypoint", and "WorkingDir" translations for Kubernetes. [7cbc607](https://github.com/Azure/iotedge/commit/7cbc607ccc483d6e0ab9642be76c8b2d8bc09605)
+* Add Experimental k8s create option feature for pod security context, resources, volumes, nodeSelector, and strategy. [cf2eba9](https://github.com/Azure/iotedge/commit/cf2eba9518a947bb09ebff5dcd6ae42f66d2d045) [23b40e1](https://github.com/Azure/iotedge/commit/23b40e1bf5e9cd04af1246b2214f0759c6446ea9)
+* Update SDK version {Microsoft.Azure.Devices.1.21.0, Microsoft.Azure.Devices.Client.1.26.0}. [5148ee7](https://github.com/Azure/iotedge/commit/5148ee70b793963f74862e047039a412a488ab73)
+* Preserve any extra properties in createOptions set by the user (NVidia support through moby) [5a6d506](https://github.com/Azure/iotedge/commit/5a6d5067c832ad85b39cc15d323442a973b3d3f7)  [a747950](https://github.com/Azure/iotedge/commit/a747950864403ee8f201bbf9e78aaf0a2c067411)
```

Comment:
remove 2nd link, which is not related to this feature.

Reply:
It's the bug fix to the introduced feature.